### PR TITLE
[COMPOSER] Added creating ruby version file and update some gems

### PR DESCRIPTION
**Pull request by MProG**

### CODE REVIEW

File: recipes/deployment.rb [hunk 1]
```diff
@@ -11,13 +11,13 @@
...
     add_gem 'sqlite3', group: [:development, :test]
-    add_gem 'pg', group: :production
+    add_gem 'pg', '>= 0.18', group: :production
   end
 end
 
 if prefer :deployment, 'capistrano3'
   say_wizard "installing gems for Capistrano"
-  add_gem 'capistrano', '~> 3.0.1', group: :development
+  add_gem 'capistrano', '>= 3.6.0', group: :development
```
Comment (AleksSenkou) [inline recipes/deployment.rb]: @MProG  `>= 3.6` says that we can update this gem to any version that will be bigger than `3.6`

what do you think about specifying major version here
```  
gem 'capistrano', '~> 3.6.0'
```
in order not to update it to `3.7` or later. updating to later version should be done by hands for such important gems

**Review by AleksSenkou — COMMENTED** [review]
@MProG looks good 👍 
some changes required

File: recipes/deployment.rb (hunk 1 above)
Comment (MProG) [reply recipes/deployment.rb]: @AleksSenkou i now how work `~>` And i think 3.6 and next 3.7 will work correctly with backward compatibility. This is not major version number. For version >= 4.x.x i agree with you but not for 3.x.x

Comment (AleksSenkou) [reply recipes/deployment.rb]: @MProG :)
`>= 3.6.0`
^ would indicate that __any__ version later than (or equal to) `3.6.0` is acceptable. There is no upper bound. 

So, gem will be updated to `4.x.x` while `bundle update` right after this version will be realized

We should update `capistrano` to major/minor versions by hands.

**Pull request merged**